src/js/components/PanelCell/PanelCell.js

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { getClassName } from '../../utils/helpers'
+
+class PanelCell extends PureComponent {
+  renderChildrenHeader = (children) => <h4 className="Panel-title">{ children }</h4>
+
+  render () {
+    const { children, isHeader, className, mods, style } = this.props
+
+    // Thoughts: The docs had to variations of a heading, can we update this or are they different use cases?
+    //  * Panel-cell & Panel-cell--header
+    //  * Panel-cell > Panel-title
```

Comment:
Oh.. yeah, this is actually two different things. `Panel-title` is for the main `Panel` title in the main `Panel-header`, the other (`Panel-cell--header`)is when a Panel cell is acting as the label or title for adjacent content:


![panel](https://user-images.githubusercontent.com/9888467/37543094-7ee86524-292e-11e8-8d44-10ee594e929e.gif)

Reply:
hmm...ok. I'll take a closer look at this and make the appropriate changes.  I'll hit you up if I have any questions.